compiler/src/dmd/frontend.h

```diff
@@ -7030,7 +7030,7 @@ struct UnionExp final
         char assocarrayliteralexp[48LLU];
         char structliteralexp[76LLU];
         char compoundliteralexp[40LLU];
-        char nullexp[25LLU];
+        char nullexp[29LLU];
```

Comment:
Not too bad, eh? Seems like the size of 12 and alignment of 4 does pay off vs. the previous 16-bytes size.